admin/admin.php

```diff
@@ -121,4 +172,27 @@ public function add_filters() {
 			$this->$obj = new $class( $this );
 		}
 	}
+	/**
+	 * Retrieve the locale according to the current language instead of the language
+	 * of the admin interface.
+	 *
+	 * @since 2.9
```

Comment:
I prefer to keep the original version `@since 2.0` as the funtion was not created for 2.9.